settei/base.py

```diff
@@ -147,6 +150,46 @@ def get_raw_value(self, obj) -> typing.Tuple[bool, object]:
                 raise ConfigKeyError(key)
         return False, value
 
+    def convert_native_type(self, value) -> object:
+        union_types = get_union_types(self.cls)
+        cls = self.cls if union_types is None else union_types
+        if isinstance(cls, type) and issubclass(cls, enum.Enum):
```

Comment:
Is it unnecessary to check `isinstance(cls, type)`?

Reply:
Because the first argument of `issubclass()` only accepts type (it will raise exception otherwise) so it must be checked before.

Reply:
Dosen't know that! Good to know it. Thanks :)